tools/xabuild/SymbolicLink.cs

```diff
@@ -21,6 +21,10 @@ public static bool Create (string source, string target)
 					}
 				} else {
 					try {
+						var sourceInfo = new UnixFileInfo (source);
```

Comment:
It's odd, it sounds like [this](https://github.com/xamarin/xamarin-android/blob/master/tools/xabuild/SymbolicLink.cs#L13) `Directory.Exists` check isn't working. Is there a test/app I can try to see this breaking?

Is there a way to do this where the symbolic link is recreated on Windows, too? Is delete directory/recreate the link the only way to do that?

Reply:
I have no idea how this would work on windows. 
I hit this by running the msbuild unit tests using MSBUILD=msbuild. 
so just using `MSBUILD=msbuild xabuild` should repo the problem.